src/test/shared/codelens/localLambdaRunner.test.ts

```diff
@@ -57,8 +59,12 @@ class FakeBasicLogger implements BasicLogger {
 
 describe('localLambdaRunner', async () => {
 
-    const tempDir: string = await fsUtils.makeTemporaryToolkitFolder()
-    ExtensionDisposableFiles.getInstance().addFolder(tempDir)
+    let tempDir: string
+    before(async () => {
+        tempDir = await fsUtils.makeTemporaryToolkitFolder()
+        await ExtensionDisposableFiles.initialize(new FakeExtensionContext())
+        ExtensionDisposableFiles.getInstance().addFolder(tempDir)
```

Comment:
In unit test runs, `ExtensionDisposableFiles` is never initialized and is not disposed when the test run completes. Suggest to clean up in `after` instead.

Reply:
✔️ (also inserted a deletion of the directory from the `makeBuildDir` test)